packages/react/src/Tabs/Tabs.tsx

```diff
@@ -12,33 +12,38 @@ import { TabsList } from './TabsList'
 import { TabsPanel } from './TabsPanel'
 import { useKeyboardFocus } from '../common/useKeyboardFocus'
 
-export type TabsProps = PropsWithChildren<HTMLAttributes<HTMLDivElement>>
-
-const TabsRoot = forwardRef(({ children, className, ...restProps }: TabsProps, ref: ForwardedRef<HTMLDivElement>) => {
-  const tabsId = useId()
-  const [activeTab, setActiveTab] = useState(0)
-  const innerRef = useRef<HTMLDivElement>(null)
-
-  const updateTab = (tab: number) => {
-    setActiveTab(tab)
-  }
-
-  // use a passed ref if it's there, otherwise use innerRef
-  useImperativeHandle(ref, () => innerRef.current as HTMLDivElement)
-
-  const { keyDown } = useKeyboardFocus(innerRef, {
-    rotating: true,
-    horizontally: true,
-  })
-
-  return (
-    <TabsContext.Provider value={{ activeTab, updateTab, tabsId }}>
-      <div {...restProps} role="tabs" ref={innerRef} onKeyDown={keyDown} className={clsx('ams-tabs', className)}>
-        {children}
-      </div>
-    </TabsContext.Provider>
-  )
-})
+export type TabsProps = {
+  /** The initial active tab. */
+  activeTab?: number
+} & PropsWithChildren<HTMLAttributes<HTMLDivElement>>
+
+const TabsRoot = forwardRef(
+  ({ activeTab, children, className, ...restProps }: TabsProps, ref: ForwardedRef<HTMLDivElement>) => {
+    const tabsId = useId()
+    const [_activeTab, setActiveTab] = useState(activeTab ?? 0)
```

Comment:
Why the underscore? It's not private if its an exported prop right?

Reply:
I wanted to call the prop `activeTab` as well so there is a clash between that and this state variable.

Reply:
Yes, starting a variable's name with an underscore often indicates it is private. But I agree that `activeTab` is the best name for the prop. Maybe use `activeTabId` internally?